### PR TITLE
triagebot.toml: add ping aliases

### LINKED ISSUE #402: Ping aliases for `@rustbot ping the_alias`
Instead of having to remember `@rustbot ping icebreakers-cleanup-crew` it would be great if we could add a memorable alias for the group so that I could e.g. write `@rustbot ping cleanup`. We would configure this by adding a list of aliases under a ping section.

### PR DESCRIPTION
**Pull request by Centril**
I think I got the syntax right.
Closes https://github.com/rust-lang/triagebot/issues/402.

r? @Mark-Simulacrum 
cc @pnkfelix

### CODE REVIEW

**Comment by spastorino** [comment]
This is great :+1:. I was just wondering if adding a lot of aliases to cleanup crew is not more confusing. Isn't with cleanup enough?.

**Comment by Centril** [comment]
I basically included some things that I would commonly think of when thinking of the group; basically things that the crew does.

File: triagebot.toml [hunk 1]
```diff
@@ -22,6 +23,7 @@ Thanks! <3
 label = "ICEBreaker-LLVM"
 
 [ping.icebreakers-cleanup-crew]
+alias = ["cleanup", "shrink", "reduce", "bisect"]
```
Comment (Mark-Simulacrum) [inline triagebot.toml]: I would personally prefer that we avoid adding all the aliases we can think of - is cleanup not enough?

Comment (Centril) [reply triagebot.toml]: There were more that I thought of... ;) ...but these were the ones which I associate the task with most closely, i.e. I associate the actual work more closely with bisecting or shrinking than with "cleanup" (which I also associate with C-cleanup, and that isn't what this is about). Imo having more aliases which aren't misleading means folks are more likely to get the bot to do what they want. (It's good to make the bot fault tolerant, e.g. `cleanups` (with the `s` at the end) would be a good extension I think.)

Comment (Mark-Simulacrum) [reply triagebot.toml]: I don't want to try to make the code itself fault tolerant in that respect, but I suppose I can buy this argument. Could you add cleanups and such as well then? r=me with that

Comment (Centril) [reply triagebot.toml]: Added "llvms" and "cleanups". We can add more variations if people make those mistakes. :)

**Pull request merged**